### ISSUE TITLE
Package-like module system

### ISSUE THREAD

**Issue by zlodes** [laravel-idea/plugin#104]
Hi!

Does it has the ability to develop modules as independent composer package? 

We are using default laravel-app-like package structure for development some laravel packages:
```
├── composer.json
├── src
│   ├── Contracts
│   ├── Concerns
│   ├── Commands
│   ├── Events
│   ├── Exceptions
│   ├── Models
│   ├── Policies
│   ├── ...
│   └── ServiceProvider.php
└── tests
    ├── Unit
    └── Integration
```

It seems like a new `Directory modules as composer.json packages` but without root project.

Important details:
- All the packages have PSR-4 autoloading defined in the `composer.json` in the package.
- `User` model and base model class are stored in the other package with known FQCN.

Examples:
- https://github.com/spatie/laravel-permission
- https://github.com/zerodahero/laravel-workflow
- ... any other Laravel package

**Comment by adelf** [comment]
So, the only difference between this package and usual laravel app is the `app` folder renamed to `src`?

**Comment by adelf** [comment]
Implemented in 3.3 version. Please try and feedback.

**Comment by zlodes** [comment]
Works fine! Thanks! ❤️
